Sunburst37/ASSEMBLY_AND_OPERATION_INFORMATION.agc

```diff
@@ -84,20 +70,20 @@
 # MP 7 - SIVB/LEM SEPARATION
 # MISSION PHASE 8 - DPS COLD SOAK
 # MP9-DPS 1 BURN
-## Page 3
+
+## Page 2
 # MISSION PHASE 11 - DPS2/FITH/APS1
 # MISSION PHASE 13 - APS2
-# MISSION PHASE 16 - RCS COLD SOAK
+# MISSION PHASE 16 - RCS COLO SOAK
 # INTEGRATION INITIALIZATION
 # ORBITAL INTEGRATION PROGRAM
 # LMP COMMAND ROUTINES
-# AS206 MISSION SCHEDULING PACKAGE
-# 206 SERVICE ROUTINES
+# AS206 MISSION SCHEOULING PACKAGE
```

Comment:
Hmmm... the PDF really does make this D and the one in "COLD SOAK" above look like Os. But in the high-resolution images provided by archive.org, they're very clearly D's. I think PDF compression might be fighting against us here. @rburkey2005, I'm not sure the PDF is good enough quality for transcription. We should probably switch back to the reduced quality image files.